test/gtest/uct/ib/test_ud.cc

```diff
@@ -208,9 +208,9 @@ UCS_TEST_SKIP_COND_P(test_ud, duplex_tx,
                      !check_caps(UCT_IFACE_FLAG_AM_SHORT)) {
     unsigned i, N = 5;
 
+    connect();
```

Comment:
why needed?
is it possible async thread will run before disable_async() is called, which would make test fail?

Reply:
we're connecting and then removing the handler for given IB completion channel FD

Reply:
otherwise, UD will set async handler after connection establishment

Reply:
is it still needed, with iface->timer_disable?